The bug on `win64`:
```
______________________ test_matplotlib_inline_on_import _______________________

    def test_matplotlib_inline_on_import():
        pytest.importorskip("matplotlib", reason="this test requires matplotlib")
        with kernel() as kc:
            cell = "\n".join(
                ["import matplotlib, matplotlib.pyplot as plt", "backend = matplotlib.get_backend()"]
            )
            _, reply = execute(cell, user_expressions={"backend": "backend"}, kc=kc)
            _check_status(reply)
            backend_bundle = reply["user_expressions"]["backend"]
            _check_status(backend_bundle)
>           assert "backend_inline" in backend_bundle["data"]["text/plain"]
E           assert 'backend_inline' in "'agg'"

..\_test_env\lib\site-packages\ipykernel\tests\test_kernel.py:327: AssertionError
=========================== short test summary info ===========================
FAILED ..\_test_env\lib\site-packages\ipykernel\tests\test_kernel.py::test_matplotlib_inline_on_import
```

 was fixed by cleaning up the windows builders on concourse I guess

--- a/recipe/meta.yaml
+++ b/recipe/meta.yaml
@@ -45,14 +45,14 @@ requirements:
     - psutil
     - pyzmq >=17
     - tornado >=6.1
-    - traitlets >=5.1.0
+    - traitlets >=5.1.0,<6.0
 
 test:
   requires:
     - curio  # [not win]
     - flaky
     - jupyter_client
-    - numpy
+    - numpy >=1.16.1  # introduced numpy.core._multiarray_umath
     - pip
     - pytest >=6.0
     - pytest-cov

--- a/recipe/meta.yaml
+++ b/recipe/meta.yaml
@@ -74,6 +74,7 @@ about:
   license: BSD-3-Clause
   license_family: BSD
   license_file: COPYING.md
+  license_url: https://github.com/ipython/ipykernel/blob/main/COPYING.md
   summary: IPython Kernel for Jupyter
   description: |
     A powerful interactive Python shell and a Jupyter kernel to work with

--- a/recipe/meta.yaml
+++ b/recipe/meta.yaml
@@ -44,7 +44,7 @@ requirements:
     - packaging
     - psutil
     - pyzmq >=17
-    - tornado >=6.1
+    - tornado >=6.1,<7.0
     - traitlets >=5.1.0,<6.0
 
 test:

